src/Mailgun/Constants/ExceptionMessages.php

```diff
@@ -18,7 +18,7 @@ class ExceptionMessages
     const EXCEPTION_GENERIC_HTTP_ERROR = 'An HTTP Error has occurred! Check your network connection and try again.';
     const EXCEPTION_MISSING_REQUIRED_PARAMETERS = 'The parameters passed to the API were invalid. Check your inputs!';
     const EXCEPTION_MISSING_REQUIRED_MIME_PARAMETERS = 'The parameters passed to the API were invalid. Check your inputs!';
-    const EXCEPTION_MISSING_ENDPOINT = "The endpoint you've tried to access does not exist. Check your URL.";
+    const EXCEPTION_MISSING_ENDPOINT = "The endpoint you've tried to access does not exist. Check if the domain matches the domain you have configure on mailgun.";
```

Comment:
Oh, That is right. Hm, lets not include the URI then. The impact of the code is so big and the win is not. 

FYI: The proper way to do this is to create a HTTPlug plugin that throws the correct exception. Much like https://github.com/KnpLabs/php-github-api/blob/master/lib/Github/HttpClient/Plugin/GithubExceptionThrower.php